### ISSUE TITLE
ListSelector issues with GradientMobileTheme

### ISSUE THREAD

**Issue by marcelofrau** [ianharrigan/haxeui#217]
Apparently the text of the ListSelector is not displaying correctly.

**Comment by ianharrigan** [comment]
Missed this somehow... Do we know why this is happening? Any thoughts on a fix?

Cheers,
Ian

**Comment by marcelofrau** [comment]
I had made some tests and have seen that if we change the ListSelector to autosize = true, it will work fine. I need to validade with you if we can use this approach or I have to try some other approach to solve this. I also checked that this problem also occur in the GradientTheme. Later today I'll check this and try to solve.

**Comment by ianharrigan** [comment]
The only issue with setting autoSize (i can forsee), is there should also be a way to define a fixed length drop down. For example, i think all these should be possible:

* Auto size based on text, label is not zero length (i belive this works)
* Auto size when there is no label text (ie, no selection)
* Fixed width, say 300px, with label
* Fixed width, say 300px, withouth a label (ie, no selection)

I seem to remember testing this things and thought the worked... but i might be way off!

**Comment by marcelofrau** [comment]
Yes, thats true. The dropdown today is calculed based on width of the component, I'll make some changes so it can have independently size control, but I'll validate with you before make any changes.
